default.nix

```diff
@@ -1,138 +1,35 @@
-{ compiler ? "ghc8107"
-, rev      ? "7a94fcdda304d143f9a40006c033d7e190311b54"
-, sha256   ? "0d643wp3l77hv2pmg2fi7vyxn4rwy0iyr8djcw1h5x72315ck9ik"
-, pkgs     ?
-    import (builtins.fetchTarball {
-      url    = "https://github.com/NixOS/nixpkgs/archive/${rev}.tar.gz";
-      inherit sha256; }) {
-      config.allowBroken = false;
-      config.allowUnfree = true;
-      overlays = [];
-    }
-, returnShellEnv ? false
-, mkDerivation ? null
-}:
-let gitignoreSrc = import (pkgs.fetchFromGitHub {
-      owner = "hercules-ci";
-      repo = "gitignore";
-      rev = "9e80c4d83026fa6548bc53b1a6fab8549a6991f6";
-      sha256 = "04n9chlpbifgc5pa3zx6ff3rji9am6msrbn1z3x1iinjz2xjfp4p";
-    }) {};
-    nix-thunk = import ./dep/nix-thunk {};
+let inputs = (import (
```

Comment:
There are now sporadic tabs in this file, could they be converted to spaces?

Reply:
Thanks for pointing this out, I've fixed it!